2020-global/llogiq.md

```diff
@@ -0,0 +1,131 @@
+**RustFest Global 2020 Speaker Introductions**
+
+**keen** is, as the name implicates  
+very keen on how he creates  
+with code many kind  
+of lisps and behind  
+all this are, well, some Rusty crates
+
+**Yousuke Onoue** will teach  
+a way the Rust language can reach  
+the web, so assemble  
+make JavaScript tremble  
+and onwards go into the breach
+
+**Tomohiro Kato** does try  
+to get a Rust-based A.I.  
+into chips to embed  
+to get stuff on the net  
+here's hope that the circuits won't fry
+
+**Aki** has the unbreakable will  
+to get Rust to the metal, to mill  
+it into shapes snappy  
+make customers happy  
+though this battle is somewhat uphill
+
+**Tarun** helpfully teaches Rust  
+to newbies who start out and must  
+learn the concepts, the tools  
+and the various rules  
+until their own experience they trust
+
+**Zac Burns** wants to serialize  
+some ideas that we all won't despise  
+into one talk to make  
+us see what it will take  
+to make code easier to realize
+
+**Jin Mingjian** uses Rust to enhance  
+some database apps' performance  
+as he breaks apart  
+the state of the art  
+to make hashtables and b-trees dance
+
+🞱🞱🞱
+
+**Nikita** makes Rust interactive  
+so if learning it is your directive  
+you won't need to fight  
+to see what's inside  
+to become a debugging detective
+
+**Aïssata Maiga** lets me know  
+how to make bots without Arduino  
+writing Rust to move  
+her bot to my groove  
+Sure there will be some cool stuff to see, no?
+
+**Daan and Diane** get us to the hype  
+Of keeping secrets in a type  
+Disallowing creation  
+of some optimization  
+that just might tell the feds what you type
+
+**Gavin and Matthijs** show how one might  
+a large project in Rust rewrite  
+start out small, let it grow  
+until stealing the show  
+from whatever was there before, right?
+
+**Vivian** wants us to be safe  
+and our code on the web to behave  
+use Rust to generate  
+code that will validate  
+risky inputs, no need to be brave
+
+Miri ist Rust's interpreter  
+And **Christian** will gladly debate'er  
+On how to bequeath  
+her the stuff underneath  
+so she can run until much later
+
+**Anastasia** plays Rust like a flute  
+or maybe a magical lute  
+to then simulate  
+things that art may create  
+and this art does really compute
+
+🞱🞱🞱
```

Comment:
What's this?

Reply:
What does "this" refer to, specifically?
If it's for the "stars" its just nicety for separating the three blocks so it's easier to scan the content.

Reply:
![image](https://user-images.githubusercontent.com/2129/102525884-30eed880-409b-11eb-9c2b-ee7063320dd6.png)

can we just use this?

```suggestion
---
```